cluster/src/main/scala/com/evolutiongaming/cluster/NodesUnreachableException.scala

```diff
@@ -9,23 +9,19 @@ class NodesUnreachableException(addresses: Nel[Address], cause: Throwable) exten
 
   override def getCause = cause
 
-  override def getMessage = {
-    addresses map { _.hostPort } match {
-      case Nel(node, Nil) => s"node $node is unreachable"
-      case nodes          => s"nodes ${ nodes mkString ", " } are unreachable"
-    }
+  override def getMessage = addresses match {
+    case Nel(node, Nil) => s"node $node is unreachable"
+    case nodes          => s"nodes ${ nodes mkString ", " } are unreachable"
   }
 }
 
 object NodesUnreachableException {
 
-  def opt(timeoutException: TimeoutException, system: ActorSystem): Option[NodesUnreachableException] = {
```

Comment:
can we make this in backward compatible way ?

Reply:
Done, please check out.